walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/handlers/OpenId4VCIRequestHandler.kt

```diff
@@ -189,8 +194,8 @@ internal class OpenId4VCIRequestHandler(
 
         return if (requirements.isEmpty()) {
             throw OpenId4VciValidationException(
-                "There is no requirement in the credential offer",
-                VerifiedIdExceptions.REQUIREMENT_MISSING_EXCEPTION.value
+                "No grants defined in credential offer.",
+                VerifiedIdExceptions.MALFORMED_CREDENTIAL_OFFER_EXCEPTION.value
```

Comment:
If requirement is missing, REQUIREMENT_MISSING_EXCEPTION is thrown to be consistent across the app and it is easier for the developer to identify the issue.